### PR TITLE
Documentation improvements

### PR DESCRIPTION
**Pull request by roback**

### CODE REVIEW

**Comment by walro** [comment]
LGTM

**Comment by roback** [comment]
and LGTM :)

File: src/Client.php [hunk 1]
```diff
@@ -65,6 +65,8 @@ public function query() {
     /**
      * Executes the given Query and returns the result
      *
+     * This method should not be called manually.
```
Comment (dentarg) [inline src/Client.php:68]: should we also add what you should do? `, instead use {Query#execute}` that is

Comment (walro) [inline src/Client.php:68]: Definitely, I just didn't know how to make the fancy reference in phpdoc and I thought this beat nothing. :)

I think we should "sync" all our API documentation at a later stage once things have settled.

Comment (dentarg) [inline src/Client.php:68]: I got the syntax from this very diff :-) (https://github.com/twingly/twingly-search-api-php/pull/11/files#diff-4528a13e17ddcea8ae7919377e58843bL33)

Comment (walro) [inline src/Client.php:68]: That was just copied from the Ruby documentation

Comment (walro) [inline src/Client.php:68]: If you look at the diff again you'll see it was removed, but I bet there's a way of doing it in phpdoc too. But as I stated above, I don't think we should change things around concerning the documentation before the clients are stable.

Comment (dentarg) [inline src/Client.php:68]: > That was just copied from the Ruby documentation

Oh, I see... yeah, when I think about it, that syntax do look like ruby :stuck_out_tongue:

**Pull request merged**